### PR TITLE
DOCSP-43570-mongosync-1.9.0

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
Add release notes for 1.9.0

See note on embedded verifier content.

## STAGING 
https://deploy-preview-470--docs-cluster-to-cluster-sync.netlify.app/release-notes/1.9/#1.9.0-release

## JIRA
https://jira.mongodb.org/browse/DOCSP-43570

### CODE REVIEW

File: source/release-notes/1.9.txt [hunk 1]
```diff
@@ -19,9 +19,36 @@ Release Notes for mongosync 1.9
 This page describes changes and new features introduced in  
 {+c2c-full-product-name+} 1.9 and the {+c2c-full-beta-program+}.
 
+1.9.0 Release 
+-------------
+
+**Upcoming**
+
+
```
Comment (ajhuh-mdb) [inline source/release-notes/1.9.txt:27]: Note: Embedded verifier content will live here once #339 is merged.

**Review by nvillahermosa-mdb — APPROVED** [review]
LGTM

**Comment by jtcovan** [comment]
Thanks Ali, this looks good to me. I just added two more points to the downstream changes doc. Sorry for the late additions:
Under general release notes,

[REP-5162](https://jira.mongodb.org/browse/REP-5162): Added a stdin disclaimer to the mongosync executable regarding verification. Users will need to accept this disclaimer before mongosync will run. 
[REP-5196](https://jira.mongodb.org/browse/REP-5196): Added --acceptDisclaimer flag to bypass the disclaimer.

@jddubois @rohshar Do we want to document the `--acceptDisclaimer` flag?

**Comment by jtcovan** [comment]
LGTM, thanks! I'm going to tag one of our leads to do a quick review.

File: source/release-notes/1.9.txt [hunk 2]
```diff
@@ -12,16 +12,46 @@ Release Notes for mongosync 1.9
...
+  disclaimer to bypass it.
+
+Optimizations: 
+
+- Increased internal timeout to reduce the occurrence of timeout failures during 
+  initialization
+
+Fixed Issues: 
+
+- Fixed a bug introduced in ``mongosync`` 1.8.0 where ``mongosync`` would crash
```
Comment (jddubois) [inline source/release-notes/1.9.txt:48]: This was already included in the 1.8.1 release notes

File: source/release-notes/1.9.txt [hunk 3]
```diff
@@ -12,16 +12,46 @@ Release Notes for mongosync 1.9
...
+
+- ``mongosync`` now supports migrating documents that have an empty timestamp 
+  values, such as Timestamp(0,0) for source clusters using MongoDB version 6.0 
+  or later. 
+
+- Added a ``stdin`` disclaimer to the ``mongosync`` executable regarding 
+  verification. Users must to accept this disclaimer before ``mongosync``  
+  can run.
+
+- Added ``--acceptDisclaimer`` flag for users that have previously read the
```
Comment (jddubois) [inline source/release-notes/1.9.txt:38]: Should this be documented in the [configuration](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/configuration/) section too?

**Review by jddubois — CHANGES_REQUESTED** [review]

File: source/release-notes/1.9.txt (hunk 3 above)
Comment (ajhuh-mdb) [reply source/release-notes/1.9.txt:38]: @jtcovan mentioned that this is being documented in #339. If not, would you mind opening up a follow-up ticket, so we can pick that up and address it?

File: source/release-notes/1.9.txt (hunk 2 above)
Comment (rohshar) [reply source/release-notes/1.9.txt:48]: Yeah I don't think we need it (even though it was included in the downstream changes)

Comment (ajhuh-mdb) [reply source/release-notes/1.9.txt:48]: Gotcha. I'll remove it!

File: source/release-notes/1.9.txt (hunk 3 above)
Comment (jtcovan) [reply source/release-notes/1.9.txt:38]: Yes, that is already in #339. Staging link at https://deploy-preview-339--docs-cluster-to-cluster-sync.netlify.app/reference/mongosync/#global-options

**Review by jtcovan — APPROVED** [review]

**Review by jddubois — APPROVED** [review]

**Review by rohshar — APPROVED** [review]

**Pull request merged**